index.html

```diff
@@ -4,6 +4,10 @@
   <title>YOUR NAME HERE</title>
 </head>
 <body>
+  <h1>Kirby Tate </h1>
+  <img src="http://giphy.com/gifs/pDVK1sjLVuMA8">
+
+  </template>
```

Comment:
template?  I've never seen this tag before....

http://i.giphy.com/14mgxYFJHXGmoo.gif